osi_streamingupdate.proto

```diff
@@ -82,7 +82,5 @@ message StreamingUpdate
     // Entities that will no longer be updated, because they are considered
     // obsolete by the sender.
     //
```

Comment:
```suggestion
    //
     \note IDs of ground truth sub-messages (e.g. moving objects) are globally unique.
```

Reply:
How about something like this to make it more clear?

Reply:
The description of StreamingUpdate states that this field just references an existing object's globally unique id.
Suggestion: "Referenced IDs are globally unique."
I agree that the "reference" part comes a bit short in this field description.

Reply:
I added your suggestions @thomassedlmayer